### DISCUSSION TITLE
Menu sizes don't work?

### DISCUSSION THREAD

**Discussion by noconsulate** [saadeghi/daisyui#2064] [Q&A]
I can put this code from the docs at the very top of a Vue3 app, right after <template> in App.vue:

```
<ul class="menu menu-xs bg-base-200 w-56 rounded-box">
  <li><a>xs item 1</a></li>
  <li><a>xs item 2</a></li>
</ul>
<ul class="menu menu-sm bg-base-200 w-56 rounded-box">
  <li><a>sm item 1</a></li>
  <li><a>sm item 2</a></li>
</ul>
<ul class="menu menu-md bg-base-200 w-56 rounded-box">
  <li><a>md item 1</a></li>
  <li><a>md item 2</a></li>
</ul>
<ul class="menu menu-lg bg-base-200 w-56 rounded-box">
  <li><a>lg item 1</a></li>
  <li><a>lg item 2</a></li>
</ul>
```

All the menus look exactly the same. This is the first time I've experienced unexpected behavior with daisyui. It works as it should in Tailwind Play. Is it something to do with Vue3 perhaps? I've tried in in two projects, the other a Nuxt3 app.

**Comment by saadeghi** [comment]
Is your `content` config on `tailwind.config.js` file is addressing to all your template files and component files?  
  Read more: https://tailwindcss.com/docs/content-configuration
